src/Dotnet.Script.Core/ScriptPublisher.cs

```diff
@@ -60,35 +80,28 @@ public void CreateExecutable(ScriptContext context, LogFactory logFactory)
             if (exitcode != 0) throw new Exception($"dotnet publish failed with result '{exitcode}'");
         }
 
-        private string CreateScriptAssembly(ScriptContext context, string tempProjectDirecory)
+        private string CreateScriptAssembly(ScriptContext context, string outputDirectory, string assemblyFileName)
         {
             try
             {
-                var emitResult = _scriptEmitter.Emit<int>(context, AssemblyName);
+                var emitResult = _scriptEmitter.Emit<int>(context);
```

Comment:
I think we can remove the `PdbStream` property from the emitResult?

Reply:
Yea I already removed it since the debug information is embedded